.github/workflows/ci.yml

```diff
@@ -16,8 +16,6 @@ jobs:
     strategy:
       matrix:
         python:
-          - '3.5'
-          - '3.6'
           - '3.7'
```

Comment:
Let's also remove `3.7` and its related tests.

Reply:
Official support for `Python 3.7` is also ending on `27-06-2023` https://devguide.python.org/versions/